spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb

```diff
@@ -108,6 +108,28 @@
       let(:disability_type) { :asthma }
 
       it_behaves_like 'includes introduction'
+
+      context 'when potential asthma-related medication is available' do
+        let(:parsed_medications_data) do
+          [{ 'status' => 'active',
+             'authoredOn' => '2009-03-25T01:15:52Z',
+             'description' => 'Hydrocortisone 10 MG/ML Topical Cream',
+             'notes' => ['Hydrocortisone 10 MG/ML Topical Cream'],
+             'dosageInstructions' => ['Once per day.', 'As directed by physician.'],
+             :flagged => false },
+           { 'status' => 'active',
+             'authoredOn' => '1998-02-12T02:15:52Z',
+             'description' => 'Dose Inhaler',
+             'notes' => ['Dose Inhaler'],
+             'dosageInstructions' => ['Once per day.', 'As directed by physician.'],
+             :flagged => true }]
```

Comment:
This threw me off when looking at the PDF, I think it would normally get sorted so the flagged is first. Non-blocking but would be nice to fix this.

Reply:
@leikkisa correct, yes, it does get sorted and should be first in the list... will add to follow up list of things. TY!

Reply:
Remember to add a test to check ordering.